### PR TITLE
refactor(diffusers_pipeline): remove unused pipeline methods 🚮

### PR DESCRIPTION
**Pull request by keturn**
There was a bunch of code in the Pipeline class that hasn't been used since the backend.generator classes for them were replaced by app.invocations.

This goes a long way to making it clearer which code in here needs to be factored out for use with SDXL or other pipelines.

## What type of PR is this? (check all applicable)

- [x] Refactor
- [ ] Feature
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update
- [ ] Community Node Submission

### CODE REVIEW

**Review by psychedelicious — APPROVED** [review]
I don't understand this area enough to review the code changes, but I did test inpainting and it still works.

File: invokeai/app/invocations/latent.py [hunk 1]
```diff
@@ -231,7 +226,6 @@ def __init__(self):
             safety_checker=None,
             feature_extractor=None,
             requires_safety_checker=False,
-            precision="float16" if unet.dtype == torch.float16 else "float32",
```
Comment (lstein) [inline invokeai/app/invocations/latent.py:234]: What problem was this causing, and is it right to remove?

File: invokeai/backend/generator/inpaint.py [hunk 2]
```diff
@@ -377,3 +377,11 @@ def postprocess_size_and_mask(self, gen_result: Image.Image) -> Image.Image:
         )
 
         return corrected_result
+
+    def get_noise_like(self, like: torch.Tensor):
+        device = like.device
+        x = torch.randn_like(like, device=device)
+        if self.perlin > 0.0:
```
Comment (lstein) [inline invokeai/backend/generator/inpaint.py:384]: We're not supporting perlin in `invokeai.app.invocations.noise`, at current. It should either be removed from here or support added back there.

**Review by lstein — COMMENTED** [review]
Nice to see all that dead code removed. I have a couple of clarification questions.

File: invokeai/app/invocations/latent.py (hunk 1 above)
Comment (keturn) [reply invokeai/app/invocations/latent.py:234]: Pipeline doesn't make use of any such `precision` argument. We're passing in an already-loaded unet here, its data type has already been set.

File: invokeai/backend/generator/inpaint.py (hunk 2 above)
Comment (keturn) [reply invokeai/backend/generator/inpaint.py:384]: Good to know.

In this PR, this `get_noise_like` method is just shifted here from its superclass img2img to make it clear there is nothing remaining using the latter.

I suggest we leave this change in functionality for the next pass.

**Review by lstein — APPROVED** [review]
Ran through text2img, img2img, inpaint and outpaint and everything seems to be working.

**Pull request merged**